README.md

```diff
@@ -10,23 +10,68 @@ code in this repository extends an integration layer provided by [DrupalExtensio
 [![Latest Stable Version](https://poser.pugx.org/drupal/tqextension/v/stable?format=flat-square)](https://packagist.org/packages/drupal/tqextension)
 [![License](https://img.shields.io/badge/license-MIT-brightgreen.svg?style=flat-square)](https://packagist.org/packages/drupal/tqextension)
 
+## Requirements
+
+* composer
+* behat
+
 ## Installation
 
-- `curl -sS https://getcomposer.org/installer | php`
-- `vim composer.json`
-```json
-{
-  "require": {
-    "drupal/tqextension": "~1.0"
-  },
-  "config": {
-    "bin-dir": "bin"
-  }
-}
-```
-- `composer install`
+To install TqExtension on an existing project:
+
+- `composer require drupal/tqextension`
```

Comment:
This avoids editing the composer.json, which is good because composer keeps track of changes to the file in the composer.lock; it's best to let composer handle that.